README.md

```diff
@@ -24,6 +24,15 @@ This library requires:
 
 This package provides a Nuke plugin that creates jobs for AWS Deadline Cloud using the [AWS Deadline Cloud client library][deadline-cloud-client]. Based on the loaded comp it determines the files required, allows the user to specify render options, and builds an [OpenJD template][openjd] that defines the workflow.
 
+### Getting Started
+```sh
+$ pip install deadline-cloud-for-nuke -t target-directory
+
+$ set NUKE_PATH=target-directory-path/deadline/nuke_submitter
+``` 
+
+After installation, within the same terminal, run `.\Nuke<version>.exe` to run Nuke. The Nuke submitter should be available in the AWS Deadline menu.
```

Comment:
Just some small suggestions for this section. Let me know your thoughts


To install the submitter manually, you can use pip.

Windows:
```batch
pip install deadline-cloud-for-nuke -t target-directory
set NUKE_PATH=%NUKE_PATH%;<target-directory>/deadline/nuke_submitter
``` 

Linux & macOS:
```bash
pip install deadline-cloud-for-nuke -t <target-directory>
export NUKE_PATH=${NUKE_PATH}:<target-directory>/deadline/nuke_submitter
```

After installation and within the same shell, run the `Nuke<version>` executable. If you want the submitter available outside of this shell, consider updating `NUKE_PATH` as an environment variable rather than a shell variable.

The Nuke submitter should now be available in the AWS Deadline menu.

Reply:
I updated README.md with separate Windows/ Linux and Mac OS and added environment variable option as suggested. Thanks for the suggestion. Thanks for the suggestion.